src/material/menu/menu.ts

```diff
@@ -549,7 +549,7 @@ export class _MatMenuBase
   providers: [{provide: MAT_MENU_PANEL, useExisting: MatMenu}],
 })
 export class MatMenu extends _MatMenuBase {
-  protected override _elevationPrefix = 'mat-mdc-elevation-z';
```

Comment:
nit: this is now the same as the legacy version so we can remove the `_elevationPrefix` and just hardcode it in the base class.

Reply:
Yeah I plan to clean this up more in a followup PR. I reverted some changes in this PR to narrow down failures in TAP